internal/cli/cmd/backup/delete.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) 2024 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+
+	http://license.coscl.org.cn/MulanPSL2
+
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+package backup
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/oceanbase/ob-operator/internal/cli/backup"
+	cmdUtil "github.com/oceanbase/ob-operator/internal/cli/cmd/util"
+	"github.com/oceanbase/ob-operator/internal/clients"
+)
+
+// NewDeleteCmd delete backup policy
+func NewDeleteCmd() *cobra.Command {
+	o := backup.NewDeleteOptions()
+	logger := cmdUtil.GetDefaultLoggerInstance()
+	cmd := &cobra.Command{
+		Use:     "delete <tenant_name>",
+		Short:   "Delete backup policy for the specified tenant",
+		Long:    "Delete backup policy for the specified tenant",
+		Args:    cobra.ExactArgs(1),
+		PreRunE: o.Parse,
+		Run: func(cmd *cobra.Command, args []string) {
+			err := clients.DeleteTenantBackupPolicy(cmd.Context(), types.NamespacedName{
+				Namespace: o.Namespace,
+				Name:      o.Name + "-backup-policy",
```

Comment:
If we want the CLI tool to be compatible with other tools, we can not assume that the name of  backup policy for a tenant is `<tenantName>-backup-policy`. For example, if users create backup policy by applying YAML manifests, the name of them may not be like this. Use labels to locate the backup policy.